src/main/java/de/oliver/fancyholograms/listeners/PlayerListener.java

```diff
@@ -40,17 +40,6 @@ public void onQuit(@NotNull final PlayerQuitEvent event) {
         });
     }
 
-    @EventHandler(priority = EventPriority.MONITOR, ignoreCancelled = true)
-    public void onMove(@NotNull final PlayerMoveEvent event) {
-        if (!event.hasChangedBlock()) {
-            return; // reduce checks we need to do
-        }
-
-        for (final Hologram hologram : this.plugin.getHologramsManager().getHolograms()) {
-            hologram.checkAndUpdateShownStateForPlayer(event.getPlayer());
-        }
-    }
-
     @EventHandler(priority = EventPriority.MONITOR)
     public void onTeleport(@NotNull final PlayerTeleportEvent event) {
```

Comment:
Is this and onWorldChange still required?

Reply:
> Is this and onWorldChange still required?

It is not necessary to remove it, since in my tests with 300 real players it does not affect the main thread at all